docs/clients/go-client-api.md

```diff
@@ -124,6 +124,16 @@ found val:  for key: Company
 
 RawKVClient is a client of the TiKV server and only supports the GET/PUT/DELETE/SCAN commands. The RawKVClient can be safely and concurrently accessed by multiple goroutines, as long as it is not closed. Therefore, for one process, one client is enough generally.
 
+### Possible Error
```

Comment:
@disksing 

I think we should update our doc to use client-go directly, not use tidb.